notebooks/official/training/pytorch_gcs_data_training.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>The pip installs seem complicated. Consider using the template for install and restart</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Which template--the notebook template? I think this is actually less complicated.</p><p> </p><p>Several libraries are required for this notebook. I think it's much cleaner and easier to read what libraries are added in this format.</p>

Reply:
<p>This is the notebook template: https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb</p><p> </p><p>Common libraries like pandas, numpy and matplotlib are pre-installed on Colab and Workbench environments. There is no need to install and update them.</p>

Reply:
<p>i will accept this form, but for the future official notebooks must use the template version.</p>